### PR TITLE
Add support for Kotlin

### PR DESCRIPTION
**Pull request by urielsalis**
Adds support for Kotlin, based on the Java version

### CODE REVIEW

**Comment by rohitpaulk** [comment]
Thanks @urielsalis! Will take a look on Monday

File: compiled_starters/kotlin/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+kotlin_redis.jar
```
Comment (rohitpaulk) [inline compiled_starters/kotlin/.gitignore:1]: @urielsalis do you think it'd be okay to use the standard .gitignore that GitHub uses? https://github.com/github/gitignore/blob/main/Java.gitignore

I can make the change, just wanted to confirm if you think this is okay. 

I saw that this and bittorrent had a different .gitignore for example, so thought it'd be worth standardizing

File: compiled_starters/kotlin/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+![progress-banner](https://codecrafters.io/landing/images/default_progress_banners/redis.png)
+
+This is a starting point for Java solutions to the
```
Comment (rohitpaulk) [inline compiled_starters/kotlin/README.md:3]: This says "Java" - not sure why that is, need to investigate

File: compiled_starters/kotlin/.gitignore (hunk 1 above)
Comment (urielsalis) [reply compiled_starters/kotlin/.gitignore:1]: For both Java and Kotlin the gitignore would look the same

The one for Java is a good starting point, but probably needs to be combined with the maven one (https://github.com/github/gitignore/blob/main/Maven.gitignore) and the Intellij one https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore (first part can be simplified to just .idea/)

**Comment by urielsalis** [comment]
@rohitpaulk should be fixed now!

**Pull request closed without merging**